packages/headless/src/controllers/facets/category-facet/headless-category-facet.ts

```diff
@@ -105,11 +291,7 @@ export function buildCategoryFacet(
     ...controller,
     facetSearch: restOfFacetSearch,
 
-    /**
-     * Toggles the specified facet value.
-     * @param selection The facet value to toggle.
-     */
-    toggleSelect: (selection: CategoryFacetValue) =>
+    toggleSelect: (selection) =>
```

Comment:
I think it's more consistent to keep the interface here, even if not needed. Maybe one other benefit is it's possible to ctrl-click to inspect the properties.